samples/bluetooth/hci_rpmsg/README.rst

```diff
@@ -0,0 +1,40 @@
+.. _bluetooth-hci-rpmsg-sample:
+
+Bluetooth: HCI RPMsg
+####################
+
+Overview
+********
+
+This sample exposes :ref:`bluetooth_controller` support
+to another device or CPU using RPMsg transport which is
+a part of `OpenAMP <https://github.com/OpenAMP/open-amp/>`__.
+
+Requirements
+************
+
+* A board with :ref:`ipm_interface` driver and Bluetooth LE support
+
+Building and Running
+********************
+
+This sample can be found under :zephyr_file:`samples/bluetooth/hci_rpmsg`
+in the Zephyr tree.
+
+To use this application, you need a board with a Bluetooth controller
```

Comment:
These sample instructions feel incomplete. Can you provide a specific board you've tested with, and show the build steps and output when running this sample?  Exactly what does the sample do?  How do you set up a separate Bluetooth HCI RPMsg peer?  You list one Kconfig option that might need adjusting, what are the others?

Reply:
This sample is supposed to work on `nrf5340_dk_nrf5340_cpunet` like _sample.yaml_ suggests in "platform_whitelist". This is the board it was tested with.
In general, this sample is really a standalone BLE Controller compatible with HCI driver and uses RPMsg transport with the IPM driver as a backend. As such, it is only functional with a counterpart BLE sample which has the BLE Host stack and uses the Bluetooth HCI driver with RPMsg transport (also part of this PR).
Does it answer your question?